Patch to fix SSL support with LibreSSL.

Taken from https://invisible-mirror.net/archives/lynx/patches/lynx2.8.9dev.17.patch.gz

See also: https://bugs.alpinelinux.org/issues/7859

diff -upr lynx2-8-8.orig/WWW/Library/Implementation/HTTP.c lynx2-8-8/WWW/Library/Implementation/HTTP.c
--- lynx2-8-8.orig/WWW/Library/Implementation/HTTP.c	2018-05-20 11:28:09.464968615 +0200
+++ lynx2-8-8/WWW/Library/Implementation/HTTP.c	2018-05-20 11:28:19.171665560 +0200
@@ -720,7 +720,7 @@ static int HTLoadHTTP(const char *arg,
 #elif SSLEAY_VERSION_NUMBER >= 0x0900
 #ifndef USE_NSS_COMPAT_INCL
 	if (!try_tls) {
-	    handle->options |= SSL_OP_NO_TLSv1;
+	    SSL_set_options(handle, SSL_OP_NO_TLSv1);
 #if OPENSSL_VERSION_NUMBER >= 0x0090806fL && !defined(OPENSSL_NO_TLSEXT)
 	} else {
 	    int ret = (int) SSL_set_tlsext_host_name(handle, ssl_host);
Only in lynx2-8-8/WWW/Library/Implementation: HTTP.c.orig
